pull_request_template.md

```diff
@@ -1,18 +1,9 @@
-<!-- Title of the PR must comply with [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) guidelines. --> 
-<!-- Title should include the JIRA ticket in square brackets after the conventional commit prefix. This will automatically link the PR in JIRA. -->
-<!-- Example: "fix: [JIRA-123] Allow creation of groups with no members" -->
-
-## Impact and Context
-<!-- Brief description of why these changes are necessary. Don't rewrite the entire Jira ticket. -->
-
-## Risks and Area of Effect
```

Comment:
I think the Impact and Context and Risks and Area of Effect headings could stay, just with the comments removed to make it cleaner.